classes/scheduledTask/ScheduledTaskHelper.inc.php

```diff
@@ -165,6 +200,7 @@ function notifyExecutionResult($id, $name, $result, $message = '') {
 	function _sendEmail($message, $subject) {
 		$mail = $this->getMail();
 		$mail->addRecipient($this->_contactEmail, $this->_contactName);
+		$mail->setFrom('bruno.beghelli@gmail.com', 'Bruno Beghelli');
```

Comment:
oups, this doesn't belongs here. :) I needed this to correctly send emails using smtp, will remove